src/packages/@ncigdc/tableModels/utils.tsx

```diff
@@ -8,14 +7,28 @@ import { DATA_CATEGORIES } from '@ncigdc/utils/constants';
 import { Th, Td, ThNum, TdNum } from '@ncigdc/uikit/Table';
 import { makeFilter } from '@ncigdc/utils/filters';
 import { findDataCategory } from '@ncigdc/utils/data';
+import { IListLinkProps } from '@ncigdc/components/Links/types';
+import { TCategoryAbbr, IDataCategory } from '@ncigdc/utils/data/types';
+interface ICreateDataCategoryColumnsProps{
+  title: string; 
+  countKey: string; 
+  Link: (props: IListLinkProps) => React.Component<any>; 
```

Comment:
I'm guessing that the props for React.Component are a composite type, of IListLinkProps and Link?